src/main/java/killua/task/Task.java

```diff
@@ -0,0 +1,86 @@
+package killua.task;
+
+/**
+ * Represents a generic task in the Killua application.
+ * This class provides basic functionalities for any type of task, including description and completion status.
+ */
+public class Task {
```

Comment:
Would you be open to considering making Task an abstract class, since you are not likely to have to instantiate it?